test/test_dataaccess.py

```diff
@@ -371,7 +374,7 @@ def test_describe_data(self):
         self.assertEqual(3, len(descriptor.data_vars))
         self.assertTrue('absorbing_aerosol_index' in descriptor.data_vars)
         self.assertEqual(3, descriptor.data_vars['absorbing_aerosol_index'].ndim)
-        self.assertEqual(('latitude', 'longitude', 'time'),
+        self.assertEqual(('time', 'latitude', 'longitude'),
```

Comment:
Just a question. It would be more useful for Cate Ap to get descriptors of datasets _after_ normalisation. So I'd expect:

```suggestion
        self.assertEqual(('time', 'lat', 'lon'),
```

No?

Reply:
This dataset is not opened as a cube, so it is not subject to normalisation in xcube.